lib/iris/fileformats/pp.py

```diff
@@ -859,6 +859,7 @@ def __init__(self, header=None):
         self.raw_lbtim = None
         self.raw_lbpack = None
         self.boundary_packing = None
+        self._index_in_structured_load_file = None
```

Comment:
This may not be necessary, could leave uninitialised.  Suspect the cost is very low.

Reply:
Yes, on balance I do think it is worth it.